plugins/ui/src/deephaven/ui/components/action_group.py

```diff
@@ -1,13 +1,220 @@
+    overflow_mode: OverflowMode | None = "wrap",
+    button_label_behavior: ButtonLabelBehavior | None = "show",
+    summary_icon: Element | None = None,
+    orientation: Orientation | None = "horizontal",
+    items: Iterable[object] | None = None,
+    disabled_keys: Iterable[str] | None = None,
+    is_disabled: bool | None = None,
+    selection_mode: SelectionMode | None = None,
+    disallow_empty_selection: bool | None = None,
+    selected_keys: SelectedKeys | Iterable[str] | None = None,
+    default_selected_keys: SelectedKeys | Iterable[str] | None = None,
+    on_action: Callable[[str], None] | None = None,
+    on_selection_change: Callable[[str], None] | None = None,
+    flex: LayoutFlex | None = None,
+    flex_grow: Number | None = None,
+    flex_shrink: Number | None = None,
+    flex_basis: DimensionValue | None = None,
+    align_self: AlignSelf | None = None,
+    justify_self: JustifySelf | None = None,
+    order: Number | None = None,
+    grid_area: str | None = None,
+    grid_row: str | None = None,
+    grid_column: str | None = None,
+    grid_column_start: str | None = None,
+    grid_column_end: str | None = None,
+    grid_row_start: str | None = None,
+    grid_row_end: str | None = None,
+    margin: DimensionValue | None = None,
+    margin_top: DimensionValue | None = None,
+    margin_bottom: DimensionValue | None = None,
+    margin_start: DimensionValue | None = None,
+    margin_end: DimensionValue | None = None,
+    margin_x: DimensionValue | None = None,
+    margin_y: DimensionValue | None = None,
+    width: DimensionValue | None = None,
+    height: DimensionValue | None = None,
+    min_width: DimensionValue | None = None,
+    min_height: DimensionValue | None = None,
+    max_width: DimensionValue | None = None,
+    max_height: DimensionValue | None = None,
+    position: Position | None = None,
+    top: DimensionValue | None = None,
+    bottom: DimensionValue | None = None,
+    left: DimensionValue | None = None,
+    right: DimensionValue | None = None,
+    start: DimensionValue | None = None,
+    end: DimensionValue | None = None,
+    z_index: Number | None = None,
+    is_hidden: bool | None = None,
+    id: str | None = None,
+    aria_label: str | None = None,
+    aria_labelledby: str | None = None,
+    aria_describedby: str | None = None,
+    aria_details: str | None = None,
+    UNSAFE_class_name: str | None = None,
+    UNSAFE_style: CSSProperties | None = None,
+) -> Element:
     """
-    An ActionGroup is a grouping of ActionButtons that are related to one another.
-
+    A contextual help is a quiet action button that triggers an informational popover.
     Args:
-        children: A list of Item or primitive elements.
-        **props: Any other ActionGroup prop.
+        *children: The children of the contextual help popover.
+        is_emphasized: An list of Item elements or a function. If the latter, a list of items must be provided using the items prop.
```

Comment:
We don't allow a function, only a bool.
```suggestion
        is_emphasized: Whether the action buttons should be displayed with emphasized style.
```